packages/ui/src/hooks/useCollection.tsx

```diff
@@ -8,9 +9,10 @@ export default function (
   query?: paths['/collection/v2']['get']['parameters']['query']
 ) {
   const [resp, setResp] = useState<CollectionResponse | null>(null)
+  const coreContext = useContext(ReservoirCoreContext)
 
   useEffect(() => {
-    if (query && ReservoirSDK.initialized) {
+    if (query && coreContext.initialized) {
```

Comment:
hmm i don't really think this is needed. why not just pass the sdk as context instead